tests/research/test_research_harness.py

```
@@ -0,0 +1,438 @@
+START_2020_MS = 1_577_836_800_000
+END_2022_MS = 1_640_995_200_000
+
+POLICY = OutcomeAccessPolicy(
+    stage="development",
+    start_inclusive_ms=START_2020_MS,
+    end_exclusive_ms=END_2022_MS,
+    allowed_years=(2020, 2021),
+)
+
+
+def _authorized(
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_authorized`

(ANN202)